commerce_coordinator/apps/lms/signals.py

```diff
@@ -18,54 +18,53 @@
 
 
 # Basic signal receiver to test out connecting signals and handlers via config
+@coordinator_receiver(logger)
 def test_receiver(sender, **kwargs):
-    logger.info(f"LMS TEST_RECEIVER CALLED with sender '{sender}'!")
+    pass
```

Comment:
This function only had a log and doesn't do anything else, not sure what else I could do here after adding the decorator to replace the log

Reply:
yeah good question! It does look a little confusing now 'as is' since it doesn't do anything- maybe either: 
a) add a comment that it's an empty example that merely logs thanks to the decorator
b) remove it now that the ones below are better examples

Reply:
I'd vote for:
> a) add a comment that it's an empty example that merely logs thanks to the decorator